### DISCUSSION TITLE
installing foundry on windows 10

### DISCUSSION THREAD

**Discussion by kwadboom** [Cyfrin/foundry-full-course-cu#2195] [Q&A]
step by step guide to install   '''curl -L https://foundry.paradigm.xyz | bash
with error message curl: (28) ssl connection timed out on ubuntu

**Comment by EngrPips** [comment]
Hello @kwadboom, This seems like an internet connection. I will suggest you try again when your network is more stable.

**Reply by kwadboom** [reply]
okay thank you i appreciate

**Reply by kwadboom** [reply]
step by step guide to install '''curl -L https://foundry.paradigm.xyz/ | bash
with error message curl: (28) ssl connection timed out on ubuntu but my issue is that it has been very consistent anytime i want to install foundry

**Reply by EngrPips** [reply]
Is this your first time installing `Foundry`

**Reply by kwadboom** [reply]
![foundry ](https://github.com/user-attachments/assets/f8134d41-fd1c-40fa-ad7f-0c145e78646a)
yes is my first time

**Comment by kwadboom** [comment]
curl -L https://foundry.paradigm.xyz | bash
% total % received % xferd average speed  time  time  time  current 
                           dload   upload total spent left  speed
100 167 100 167    0 0    268      0     --:-- --:-- --:--  268
0    0   0   0     0 0     0       0     --:-- --:-- --:--   0
curl: (28) SSL connection timeout

**Reply by EngrPips** [reply]
This still seems like an internet connection problem. Are you confident that your internet connection is stable and fast?
